tests/Unit/CategoryTest.php

```diff
@@ -51,12 +51,20 @@ public function it_has_type()
         $this->assertEquals(Category::EXPENSES, $sut->type);
     }
 
+    /** @test */
+    public function it_has_color()
+    {
+        $sut = Category::factory()->create(['color' => 'badge-gray']);
```

Comment:
same as mentioned above, let's just store the color itself without the `badge-` prefix